vendor/brave-ios/BATBraveRewards.mm

```diff
@@ -104,6 +105,7 @@ - (instancetype)initWithConfiguration:(BATBraveRewardsConfiguration *)configurat
 
     BATBraveAds.debug = configuration.environment != BATEnvironmentProduction;
     BATBraveAds.environment = configuration.environment;
+    BATBraveAds.buildChannel = configuration.buildChannel;
```

Comment:
`configuration.buildChannel` will be `nil` by default if client doesn't set it, which will make `isRelease` false and the name an empty string. Is this an acceptable input?

Reply:
Fixed